go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/algorand/avm-abi v0.2.0
-	github.com/algorand/go-algorand-sdk/v2 v2.0.0-20230228201805-5b8c99b1412c
+	github.com/algorand/go-algorand-sdk/v2 v2.0.0-20230511222239-d0007f96cf95
```

Comment:
TODO: Do not merge as is ^^^. Only merge after merging https://github.com/algorand/go-algorand-sdk/pull/531 and updating with `go get go-algorand-sdk`.